### DISCUSSION TITLE
ERROR: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by FelipeMSD** [nextflow-io/nextflow#4552] [Q&A] [answered]
Dear all,

I've been trying to run this software: [PRROMenade](https://github.com/ComputationalGenomics/PRROMenade/tree/master), with command line `./taxonomic-classifier -i <folder_with_index/index_prefix> <read_file> -t <num_proc> -o <out-bin-file>`  as part of a Nexflow process, however, I had this error "**prromenade-classifier: error while loading shared libraries: libz.so.1: cannot open shared object file: No such file or directory**". Even after I tried many containers with package [zlib](https://www.zlib.net), such as [biocontainers/bioconductor-zlibbioc](https://quay.io/repository/biocontainers/bioconductor-zlibbioc?tab=tags) Nextflow keeps throwing me the same error, which does not happen if i run the tool without Nextflow... Do you have any tips on this?

**Comment by edmundmiller** [comment, marked as answer]
https://community.seqera.io/t/libz-so-missing-for-aws-cli-in-aws-batch-tasks/1869/4?u=emiller

In case anyone finds this in the future!
